lighthouse-core/audits/byte-efficiency/uses-optimized-images.js

```diff
@@ -41,7 +40,7 @@ class UsesOptimizedImages extends Audit {
     return {
       category: 'Images',
       name: 'uses-optimized-images',
-      description: 'Has optimized images',
+      description: 'Avoids unoptimized images',
```

Comment:
Any thoughts on CSS wording changes to also be affirmative? I'll give up soon if you don't have any ideas either and just live with the mismatch to push this through, haha :P